### DISCUSSION TITLE
How to Debug Flutter app wirless in windows

### DISCUSSION THREAD

**Discussion by Floyden-Monteiro** [community/community#133484] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

How to Debug Flutter app wirless in windows

**Comment by DeveloperDrogo** [comment, marked as answer]
Enable Developer Options and USB Debugging:

On your Android device, go to Settings > About phone.
Tap Build number seven times to enable developer mode.
Go back to Settings > System > Developer options.
Enable USB debugging.
Connect Device via USB:

Connect your Android device to your computer using a USB cable.
